core-plugins/src/main/java/co/cask/hydrator/plugin/common/CubeUtils.java

```diff
@@ -41,15 +41,15 @@ private CubeUtils() {
       if (split.length != 2) {
         //should not happen
         throw new IllegalArgumentException(
-          String.format("For Property {} Parameter {} , name or value is empty, please check config", property, group));
+          String.format("For Property %s Parameter {}, name or value is empty, please check config", property, group));
```

Comment:
also replace the second {}. Also I feel this message doesn't seem to make sense, `split.length != 2` may not just mean name or value is empty. Also is `group` a parameter?

Reply:
I'm not sure if group is a parameter, but I replaced the second `{}` now.